docs/schedule/examples/fares-v2.md

```diff
@@ -216,7 +216,167 @@ The single ride fare product shown below has both `cash` and `tap-to-ride` fare
+|-----------|---------|
+| mnr_1     | ITO1887 |
+| mnr_1     | ITO2383 |
+| mnr_HUD-5 | ITO1804 |
+| mnr_HUD-6 | ITO1669 |
+| mnr_HUD-6 | ITO1824 |
+| mnr_HUD-7 | ITO1856 |
+| mnr_HUD-7 | ITO1897 |
+| mnr_HUD-8 | ITO1777 |
+| mnr_HUD-8 | ITO1789 |
+| mnr_HUD-9 | ITO2096 |
+
+
+Service days for train services 3 and 13 are defined using calendar.txt. Notably, general service days (i.e. weekdays, weekends and anyday) are also defined, as these will be associated with specific timeframes in order to model `time-variable fares`.
+
+[**calendar.txt**](../../reference/#calendartxt)
+
+| service_id | monday | tuesday | wednesday | thursday | friday | saturday | sunday | start_date | end_date |
+|------------|--------|---------|-----------|----------|--------|----------|--------|------------|----------|
+| 13         | 1      | 1       | 1         | 1        | 1      | 0        | 0      | 20230612   | 20231006 |
+| 3          | 1      | 1       | 1         | 1        | 1      | 0        | 0      | 20230609   | 20231006 |
+| weekdays   | 1      | 1       | 1         | 1        | 1      | 0        | 0      | 20220101   | 20240101 |
+| weekends   | 0      | 0       | 0         | 0        | 0      | 1        | 1      | 20220101   | 20240101 |
+| anyday     | 1      | 1       | 1         | 1        | 1      | 1        | 1      | 20220101   | 20240101 |
+
+
+Specific time frames are created in `timeframes.txt`, including general service days (anytime, weekdays and weekends), and peak and off peak periods:
+
+* AM Peak: from 6 am to 10 am on weekdays
+* AM2PM Peak: from 6 AM to 9 AM and from 4 pm to 8 pm on weekdays
+* Not AM Peak: weekday time not included in AM Peak
+* Not AM2PM Peak: weekday time not included in AM2PM Peak
+
+[**timeframes.txt**](../../reference/#timeframestxt)
+
+| timeframe_group_id | start_time | end_time | service_id |
+|:------------------:|:----------:|:--------:|:----------:|
+|       anytime      |  00:00:00  | 24:00:00 |   anyday   |
+|      weekdays      |  00:00:00  | 24:00:00 |  weekdays  |
+|      weekends      |  00:00:00  | 24:00:00 |  weekends  |
+|     mnr_ampeak     |  06:00:00  | 10:00:00 |  weekdays  |
+|    mnr_notampeak   |  00:00:00  | 06:00:00 |  weekdays  |
+|    mnr_notampeak   |  10:00:00  | 24:00:00 |  weekdays  |
+|    mnr_am2pmpeak   |  06:00:00  | 09:00:00 |  weekdays  |
+|    mnr_am2pmpeak   |  16:00:00  | 20:00:00 |  weekdays  |
+|  mnr_notam2pmpeak  |  00:00:00  | 06:00:00 |  weekdays  |
+| mnr_notam2pmpeak   | 09:00:00   | 16:00:00 | weekdays   |
+| mnr_notam2pmpeak   | 20:00:00   | 24:00:00 | weekdays   |
+
+
+Each individual fare product is defined in `fare_products.txt`, in this example, only trips between zone 1 and 9 are listed. The full dataset would include a record for each  price defined by a time and zone combination. Additionally, the example only uses one fare media (i.e. paper ticket) but additional combinations could be created if prices would also vary depending on fare media.
```

Comment:
Suggestion

Each individual fare product is defined in `fare_products.txt`. In this example, only trips between zone 1 and 9 are listed. The full dataset would include a record for each price defined by a time and zone combination. Additionally, the example only displays one fare media (`paper`) but additional combinations could be created if prices would also vary based on the fare media.